### ISSUE TITLE
Results view does not adjust well to zoom in and zoom out

### ISSUE THREAD

**Issue by coquagli** [Microsoft/vscode-mssql#683]
Steps to Reproduce:
1. Execute a query
2. Use Ctrl + and Ctrl - to change the zoom level

It looks like we do not react well to changes to the zoom level, decreasing ease of usability. 

Default zoom level:
![image](https://cloud.githubusercontent.com/assets/24420239/22708108/fb734006-ed29-11e6-9420-6fa4063e3275.png)

Zoomed in 2 levels:
![image](https://cloud.githubusercontent.com/assets/24420239/22708161/32c064da-ed2a-11e6-861f-b00c2ccfc907.png)

Zoomed out 2 levels: 
![image](https://cloud.githubusercontent.com/assets/24420239/22708144/1c39fc8a-ed2a-11e6-8418-6dd9724dc5c8.png)

**Comment by kevcunnane** [comment]
This is a duplicate of #280 - we're blocked on Chromium issues with a WebView which require a fix in Chromium, then updates to Chromium (in Electron) and Electron (in VSCode). It's likely this will be around for a while unfortunately. Closing since we have a bug tracking it.
